plugins/tomcat.yaml

```diff
@@ -19,7 +19,7 @@ parameters:
     label: Pod Name
     description: The pod name (without the unique identifier on the end)
     type: string
-    required: true
+    default: 'tomcat-*'
```

Comment:
Shouldn't we also be setting the default using the go template?

Reply:
I agree. Added defaults.